src/structures/command/Command.js

```diff
@@ -148,4 +148,18 @@ module.exports = class Command {
       subcommands
     }
   }
+
+  getEmoji (emoji) {
+    const emojiGuilds = process.env.EMOJI_GUILDS && process.env.EMOJI_GUILDS.split(',')
+    if (!emojiGuilds) return '❓'
+
+    const results = emojiGuilds.map(eg => {
+      const matchingEmoji = this.client.emojis.filter(e => e.guild.id === eg && e.name.toLowerCase() === emoji.toLowerCase()).first()
+      if (matchingEmoji) return matchingEmoji.toString()
+      else return null
+    })
+
+    if (results.filter(r => r)) return results.filter(r => r)[0]
```

Comment:
Use Boolean instead of and put this filter on variable.
Code will look like this:
```js
const filter = results.filter(Boolean);
if (filter) return filter[0];
```